lib/openapi/revisions/est-spec-1.0.0-20231214.json

```diff
@@ -0,0 +1,1536 @@
+          },
+          "404": {
+            "description": "Error PhotoniqEST component settings not found"
+          },
+          "500": {
+            "description": "Error getting all PhotoniqEST component settings"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Delete all PhotoniqEST component settings",
+        "responses": {
+          "200": {
+            "description": "All PhotoniqEST component settings deleted successfully"
+          },
+          "500": {
+            "description": "Error deleting all PhotoniqEST component settings"
+          }
+        }
+      }
+    },
+    "/api/est/v1/components/settings/{component}": {
+      "get": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Get PhotoniqEST component setting",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "component",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Name of the component"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns PhotoniqEST component setting"
+          },
+          "404": {
+            "description": "Error PhotoniqEST component setting not found"
+          },
+          "500": {
+            "description": "Error getting PhotoniqEST component setting"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Update PhotoniqEST component setting",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "component",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Name of the component"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns updated PhotoniqEST component setting"
+          },
+          "500": {
+            "description": "Error updating PhotoniqEST component setting"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Delete PhotoniqEST component setting",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "component",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Name of the component"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "PhotoniqEST component setting deleted successfully"
+          },
+          "500": {
+            "description": "Error deleting PhotoniqEST component setting"
+          }
+        }
+      }
+    },
+    "/api/est/v1/error/hourly": {
+      "get": {
+        "tags": [
+          "Error Metrics"
```

Comment:
Could these metrics be listed next to Usage Metrics? It makes sense to group similar items together.